src/components/pagination/Pagination.vue

```diff
@@ -134,6 +137,13 @@ export default {
             return parseInt(this.rangeAfter)
         },
 
+        newIconPrev() {
+            return this.iconPrev || config.defaultIconPrev
```

Comment:
Why do we need this ? I think you might add  config as default value of the prop (so in other components)

Reply:
Sure!
It was done this way in some other components.

Fixed!